contracts/UFragments.sol

```diff
@@ -227,16 +227,16 @@ contract UFragments is ERC20Detailed, Ownable {
 
     /**
      * @dev Function to check the amount of tokens that an owner has allowed to a spender.
-     * @param owner The address which owns the funds.
+     * @param owner_ The address which owns the funds.
      * @param spender The address which will spend the funds.
      * @return The number of tokens still available for the spender.
      */
-    function allowance(address owner, address spender)
+    function allowance(address owner_, address spender)
```

Comment:
Should we change owner to sender for clarity?

Reply:
Not sure if `sender` would make sense. 

The parameter refers to the owner of the token. So potentially `wallet-x` could check how much allowance `wallet-y` sets for `contract-x` to spend on x's behalf.

https://theethereum.wiki/w/index.php/ERC20_Token_Standard

@brandoniles ?

Reply:
sender meaning the token sender not the transaction sender. We can rename to tokenOwner as the wiki example. Owner is mildly confusing with contract owner.

Reply:
👍

Reply:
Sender would be confusing I think.

Owner conforms to the convention in the ERC20 standard:
https://theethereum.wiki/w/index.php/ERC20_Token_Standard

Zeppelin, the interface we implement, also uses owner in this context and in the ownable context with no issues.

Reply:
Okay then. Sticking to `owner` for now, we can revisit this later if necessary.

Reply:
@brandoniles I recommend tokenOwner over owner as standard here https://theethereum.wiki/w/index.php/ERC20_Token_Standard